### DISCUSSION TITLE
1608チップ抵抗の入手方法

### DISCUSSION THREAD

**Discussion by matsujirushi** [algyan/XIAOGYAN#23] [Q&A] [answered]
秋月電子通商で、1608チップ抵抗をバラ売りしていなかった...
（いまになって気づいた）

どうするのが良いんだろう？

案1. チップ抵抗をリード抵抗に変更する（アートワーク辛い）
案2. 秋月電子通商からリールで買う（狙った金額を大幅にオーバー）
案3. チップ抵抗だけ他店から買う（手間と送料が増える）

**Comment by tatsu1225** [comment]
案1 の改編で，リード抵抗用のパッドに変更．
チップ抵抗を持っている / 手に入りやすい人は，以下のイメージではんだ付け
https://toragi.cqpub.co.jp/Portals/0/backnumber/2010/07/p084.pdf

リード抵抗の人は，縦にリード抵抗を曲げて実装というのはどうでしょう？
(言ってて，だいぶ苦肉ですね．イマイチかも

**Comment by tatsu1225** [comment]
100kは1608mmのバラ売り(5個入・100円)あるようですけど，10kが1608mmないんですね
10kの2012はあるようですけど，こちらだとイマイチですか？
  10k :  https://akizukidenshi.com/catalog/g/gR-11797/
  100k : https://akizukidenshi.com/catalog/g/gR-11792/

0Ωはショートランドで対応？

**Comment by y-ohe** [comment]
千石電商だとチップ抵抗は10個、50個、リール(5000個)と、ほぼE24系列で揃っているから集めやすいんですよね。
秋月しばりにしたいところですが、チップ抵抗は千石電商も選択肢に入れるものありですかねー
もしくは、ちょっと手間がかかって大変なんですが、基板とセットでプレゼントしちゃうとか。
BOMを見ると、今のところ0, 10k, 100kとR7の100 ～ 1kあたりの4種類ぐらいみたいなので、リールからそれぞれ10個ずつにカットして基板と一緒に宅配する感じですね。

**Comment by matsujirushi** [comment, marked as answer]
みなさん、情報ありがとうございます。
以下、結論です。

* 抵抗は1608チップ抵抗のまま。
* 使用する抵抗は10Kの1種類にする。ES3で対応済み。（0も使用するが、はんだショートでも代用できる。どちらにするかは組立手順つくる人に一任。）
* 周年記念イベントでは、基板配布時に10Kを同封。（カットと梱包、小暮さんに快諾いただきました！ありがとうございます！）
